### DISCUSSION TITLE
Send Release Notes document through mail

### DISCUSSION THREAD

**Discussion by AshishNigamNoida** [rfennell/AzurePipelines#1089] [General]
Hi,

Is there any reusable feature to send the release note (.md) file through mail to stakeholders?

Please let me know.

Regards,
Ashish

**Comment by rfennell** [comment]
There is nothing built into my task as it would be scope creep. You need to use another task from the marketplace, there appear to be a few for this type of job, but it is not something I have tried so cannot recommend one. 

Rather we upload our release notes to a WIKI as a permanent record

See my video on the subject https://www.youtube.com/embed/xaV3dFoQdV8?start=69 and the associate blog post https://blogs.blackmarble.co.uk/rfennell/2020/09/15/how-can-i-automatically-create-azure-devops-release-notes-and-how-can-i-publish-them/
